Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM ubuntu:14.04.4
+FROM ubuntu:14.04
 
-# Replace shell with bash so we can source files
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
```

Comment:
This line guarantees that when you run an exec command, that we are using the bash shell and not the minimal dash shell. Bash is necessary for things like properly sourcing in environment vars.